src/main/resources/com/google/jenkins/plugins/computeengine/InstanceConfiguration/config.jelly

```diff
@@ -35,15 +32,6 @@
             <f:entry title="${%Number of Executors}" field="numExecutorsStr">
                 <f:textbox default="1"/>
             </f:entry>
-            <f:entry title="${%Run as user}" field="runAsUser">
-                <f:textbox default="${descriptor.defaultRunAsUser()}"/>
-            </f:entry>
-            <f:entry title="${%Remote Location}" field="remoteFs">
-                <f:textbox default=""/>
-            </f:entry>
-            <f:entry title="${%Java Exec Path}" field="javaExecPath">
-                <f:textbox default="java"/> 
-            </f:entry>
             <f:entry field="windows" title="${%Windows?}">
```

Comment:
Im not sure if windows fields should not be in launch config too..

Reply:
I think you're right. I was originally thinking it is more of a property of the instance, but these three fields specifically determine parts of the launch process, while the actual instance property is the image used.